src/re_frame/interceptor.cljc

```diff
@@ -192,7 +216,9 @@
    already done.  In advanced cases, these values can be modified by the
    functions through which the context is threaded."
   [event-v interceptors]
-  (-> (context event-v interceptors)
-      (invoke-interceptors :before)
-      change-direction
-      (invoke-interceptors :after)))
+  (if-let [error-handler (registrar/get-handler :error :event-handler)]
```

Comment:
My opinion: I wouldn't allow to register multiple handlers. At least not until someone asks for it. Seems like in most cases users should be able to compose multiple handlers into one.

Reply:
Good call about catching errors in subscriptions, I didn't have the need to do that but I can see it might be useful. Do you foresee a unified approach covering exceptions in interceptors and subscriptions?

Reply:
I think a `reg-sub-error-handler` should be fine. No need to couple it with this system, though it probably would work the same way.